src/test/components/MarkerTable.test.js

```diff
@@ -147,9 +148,55 @@ describe('MarkerTable', function() {
     expect(getRowElement('foobar')).toHaveClass('isRightClicked');
   });
 
+  it("can copy a marker's cause using the context menu", () => {
+    jest.useFakeTimers();
+
+    // This is a tid we'll reuse later.
+    const tid = 4444;
+
+    // Just a simple profile with 1 thread and a nice stack.
+    const {
+      profile,
+      funcNamesDictPerThread: [{ E }],
+    } = getProfileFromTextSamples(`
+      A[lib:libxul.so]
+      B[lib:libxul.so]
+      C[lib:libxul.so]
+      D[lib:libxul.so]
+      E[lib:libxul.so]
+    `);
+    profile.threads[0].name = 'Main Thread';
+
+    // Add another thread with a known tid that we'll reuse in the marker's cause.
+    profile.threads.push(getEmptyThread({ name: 'Another Thread', tid }));
+    // Add the reflow marker to the first thread.
+    addMarkersToThreadWithCorrespondingSamples(profile.threads[0], [
+      getReflowMarker(3, 100, {
+        tid: tid,
+        // We're cheating a bit here: E is a funcIndex, but because of how
+        // getProfileFromTextSamples works internally, this will be right right
```

Comment:
nit: extra "right". I guess one "right" should be enough :)

Reply:
ah "right", I wanted "the right" :) thanks!